spec/integration/project_spec.rb

```diff
@@ -6,14 +6,14 @@
       project1 = create(:project)
       project2 = create(:project, title: "Java Tic-Tac-Toe")
 
-      visit "/"
+      visit "/projects"
```

Comment:
We should probably just get rid of this index route, you think? We won't actually be using it so I don't think people should be able to navigate to it.